unit-tests/src/test/java/org/eclipse/collections/impl/factory/ArraysTest.java

```diff
@@ -0,0 +1,26 @@
+/*
+ * Copyright (c) 2017 BNY Mellon.
+ * All rights reserved. This program and the accompanying materials
+ * are made available under the terms of the Eclipse Public License v1.0
+ * and Eclipse Distribution License v. 1.0 which accompany this distribution.
+ * The Eclipse Public License is available at http://www.eclipse.org/legal/epl-v10.html
+ * and the Eclipse Distribution License is available at
+ * http://www.eclipse.org/org/documents/edl-v10.php.
+ */
+
+package org.eclipse.collections.impl.factory;
+
+import org.eclipse.collections.impl.list.fixed.ArrayAdapter;
+import org.junit.Assert;
+import org.junit.Test;
+
+public class ArraysTest
+{
+    @Test
+    public void adapt()
+    {
+        ArrayAdapter<String> adapted = Arrays.asList("1", "2", "3");
+        Assert.assertEquals(java.util.Arrays.asList("1", "2", "3"), adapted);
```

Comment:
Both the class and the method are named the same as the ones in java.util.Arrays.asList().